service-commands/scripts/bake-dev-image.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# bakes from latest master on gcp by default
+
+set -x
+
+# error handling
+trap handle_provisioning_error 1
+handle_provisioning_error () {
+    "Error while provisioning - make sure cloud platform and service-commands A are set up. See service-commands README for details."
+    exit 1
+}
+
+# Prerequisites:
+    # service-commands installed w A alias
+    # GCP access (if baking w GCP)
+    # Azure access (if baking w Azure)
+# Default values of arguments
+USE_AZURE=0 # GCP by default
+SOURCE_DISK="christine-new-bake" # name of remote box to bake as AMI
+PROTOCOL_GIT_REF="master"
+CLIENT_GIT_REF="master"
+USERNAME="ubuntu"
+d=`date +%m-%d-%Y`
+echo $d
+BUILD_INSTANCE_NAME="audius-bake-dev-$d"
+
+process_arguments () {
+    for arg in "$@"
+    do
+        case $arg in
+            -a|--azure)
+            USE_AZURE=1
+            shift # Remove --azure from processing
```

Comment:
Is it missing a `shift`? I rarely use `shift` just spotted a difference.

Reply:
good question! it's not missing - it's to flip a switch; `azure=1` when `--azure`; not `--azure=true`

Reply:
Ah, you're consuming just one parameter, not a parameter and a value?

Thanks! :)

Reply:
Yes! I don't know if it's cleaner and more intuitive if it follows the rest but also I don't think we're intending people to run these scripts (it should be other scripts/in CI) so we might not have to think that hard about the interface. I'm happy to change it if you think it's better to align w the other arguments though!

Reply:
Oh no, not at all. Leave it as is.

I was just asking for my own learning. Thanks for sharing!